pkgs/pkgs-lib/tests/formats.nix

```diff
@@ -1005,4 +1005,51 @@ runBuildTests {
       </root>
     '';
   };
+
+  PlistGenerate = shouldPass {
+    format = formats.plist { };
+    input = {
+      null = null;
+      false = false;
+      true = true;
+      int = 10;
+      float = 3.141;
+      str = "foo";
+      attrs.foo = 0;
+      list = [
+        1
+        2
+      ];
+      path = ./testfile;
+    };
+    expected = ''
+      <?xml version="1.0" encoding="UTF-8"?>
+      <!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+      <plist version="1.0">
+      <dict>
+      	<key>attrs</key>
+      	<dict>
+      		<key>foo</key>
+      		<integer>0</integer>
+      	</dict>
+      	<key>false</key>
+      	<false/>
+      	<key>float</key>
+      	<real>3.141000</real>
+      	<key>int</key>
+      	<integer>10</integer>
+      	<key>list</key>
+      	<array>
+      		<integer>1</integer>
+      		<integer>2</integer>
+      	</array>
+      	<key>path</key>
+      	<string>/Users/prince213/Projects/nixpkgs/pkgs/pkgs-lib/tests/testfile</string>
```

Comment:
Is this correct? I thought the file would end up in the nix store. If this is really the case, I don't think this would work on anyone elses machine

Reply:
Oops, my bad.  I'll use interpolation instead.